api/src/pcapi/scripts/algolia_indexing/indexing.py

```diff
@@ -25,6 +26,40 @@ def batch_indexing_offers_in_algolia_from_database(
         page += 1
 
 
+def batch_indexing_collective_offers_in_algolia_from_database(
+    ending_page: int = None, limit: int = 10000, starting_page: int = 0
+) -> None:
+    page = starting_page
+    while True:
+        if ending_page and ending_page == page:
+            break
+
+        offer_ids = collective_offers_repository.get_paginated_active_collective_offer_ids(limit=limit, page=page)
+        if not offer_ids:
+            break
+        search.reindex_offer_ids(offer_ids)
+        logger.info("[ALGOLIA] Processed %d collective offers from page %d", len(offer_ids), page)
+        page += 1
+
+
+def batch_indexing_collective_offers_template_in_algolia_from_database(
+    ending_page: int = None, limit: int = 10000, starting_page: int = 0
+) -> None:
+    page = starting_page
+    while True:
+        if ending_page and ending_page == page:
+            break
+
+        offer_ids = collective_offers_repository.get_paginated_active_collective_offer_template_ids(
+            limit=limit, page=page
+        )
+        if not offer_ids:
+            break
+        search.reindex_offer_ids(offer_ids)
```

Comment:
Idem plus haut, mauvaise fonction : 
 ```suggestion
        search.async_index_collective_offer_template_ids(offer_ids)
```